src/compiler/crystal/codegen/debug.cr

```diff
@@ -49,29 +59,44 @@ module Crystal
     end
 
     def get_debug_type(type)
+      get_debug_type(type, type)
+    end
+
+    def get_debug_type(type, orig_type : Type)
```

Comment:
Can we use `original_type` instead? We try to avoid abbreviations when possible (easier to read in your mind).

Reply:
Also, can we put this overload **after** all of the others? It's easier to understand if more specific overloads come first.

Reply:
^ Nevermind the last comment, I got confused with `create_debug_type`.

Reply:
Will do.